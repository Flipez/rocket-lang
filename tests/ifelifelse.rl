i = 0
while (i<3)
  if (i==0)
    puts("if/elif/el: if block")
  elif (i==1)
    puts("if/elif/el: elif block")
  else
    puts("if/elif/el: else block")
  end

  i = i+1
end
