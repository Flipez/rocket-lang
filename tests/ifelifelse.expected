"if/elif/el: if block"
"if/elif/el: elif block"
"if/elif/el: else block"
nil
